.env.example

```diff
@@ -11,3 +11,7 @@ PORT=8080
 
 # One of: trace, debug, info, warn, error, fatal, silent
 LOG_LEVEL=debug
+
+# Nodemailer config
+NODEMAILER_USER="no-reply@senecacollege.ca"
```

Comment:
Let's rename this to `NOTIFICATIONS_EMAIL_USER`.  The nodemailer bit is an internal detail that admins of the system won't care about. Add a comment above it that indicates what it's for.

Reply:
@humphd resolved in latest commit.